packages/design-picker/src/types.ts

```diff
@@ -76,7 +76,8 @@ export type DesignType =
 	| 'premium'
 	| 'standard' // The design is free.
 	| 'default' // The default design and it means user skipped the step and didn't select any design.
-	| 'anchor-fm';
+	| 'anchor-fm'
+	| 'assembler';
```

Comment:
[nit] Maybe `site-assembler`

Reply:
Hmm, I'm just following the P2

<img width="349" alt="image" src="https://user-images.githubusercontent.com/1525580/209283062-1f7269bd-400f-4f1f-9ed8-747450f0a3fe.png">

Reply:
oh okay 👍